__tests__/app.test.js

```diff
@@ -170,6 +170,82 @@ describe("app", () => {
       });
   });
 
+  describe('PATCH /api/reviews/:review_id', () => {
+    it('200: responds with whole updated review', () => {
+      return request(app)
+        .patch('/api/reviews/2/')
+        .send({
+          inc_votes: 3
+        })
+        .expect(200)
+        .then(({ body: { review } }) => {
+          expect(review).toEqual({
+            "category": "dexterity",
+            "created_at": "2021-01-18T10:01:41.251Z",
+            "designer": "Leslie Scott",
+            "owner": "philippaclaire9",
+            "review_body": "Fiddly fun for all the family",
+            "review_id": 2,
+            "review_img_url": "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+            "title": "Jenga",
+            "votes": 8
+          });
+        });
+    })
+    it('200: responds with unmodified review if empty input', () => {
+      return request(app)
+        .patch('/api/reviews/2/')
+        .send({})
+        .expect(200)
+        .then(({ body: { review } }) => {
+          expect(review).toEqual({
+            "category": "dexterity",
+            "created_at": "2021-01-18T10:01:41.251Z",
+            "designer": "Leslie Scott",
+            "owner": "philippaclaire9",
+            "review_body": "Fiddly fun for all the family",
+            "review_id": 2,
+            "review_img_url": "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+            "title": "Jenga",
+            "votes": 5
+          });
+        });
+    })
+    it('200: ignores additional request body properties', () => {
+      return request(app)
+        .patch('/api/reviews/2/')
+        .send({
+          inc_votes: 2,
+          isGroot: true
+        })
+        .expect(200)
+        .then(({ body: { review } }) => {
+          expect(review).toEqual({
+            "category": "dexterity",
+            "created_at": "2021-01-18T10:01:41.251Z",
+            "designer": "Leslie Scott",
+            "owner": "philippaclaire9",
+            "review_body": "Fiddly fun for all the family",
+            "review_id": 2,
+            "review_img_url": "https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700",
+            "title": "Jenga",
+            "votes": 7
+          });
+        });
+    })
+    it('400: wrong format eg votes NaN', () => {
+      return request(app)
+        .patch('/api/reviews/2/')
+        .send({
+          inc_votes: 'bananas'
+        })
+        .expect(400)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe('Bad Request');
+        });
+    })
+  });
+
```

Comment:
We just need to add two more tests here - 
Status 400, invalid ID, e.g. string of "not-an-id" in patch path
Status 404, non existent ID, e.g. 0 or 9999 in patch path